### PR TITLE
Fix json conversation of strings with single quote

### PR DESCRIPTION
**Pull request by esraay**
Short description (with a reference to an issue).

**DoD**

- [ ] Major project work has a corresponding task. If there’s no task for what you are doing, create it. Each task needs to be well defined and described.

- [ ] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.

- [ ] No new error or warning messages are introduced.

- [ ] All interaction with a semantic functions, examples and docs are written from the perspective of the person using or receiving it. They are understandable and helpful to this person.

- [ ] If the change affects code or repo sctructure, README, documentation and code comments should be updated. 

- [ ] All code has been peer-reviewed before merging into any main branch.

- [ ] All changes have been merged into the main branch we use for development (develop).

- [ ] Continuous integration checks (linter, unit tests) are configured and passed.

- [ ] Unit tests added for all new or changed logic.

- [ ] All task requirements satisfied. The reviewer is responsible to verify each aspect of the task.

- [ ] Any added or touched code follows our style-guide.

### CODE REVIEW

**Comment by esraay** [comment]
Closes #383

File: R/checkbox.R [hunk 1]
```diff
@@ -197,9 +197,9 @@ multiple_checkbox <- function(input_id, label, choices, choices_value = choices,
 update_multiple_checkbox <- function(session = getDefaultReactiveDomain(),
                                      input_id, choices = NULL, choices_value = choices,
                                      selected = NULL, label = NULL) {
-  if (!is.null(selected)) value <- jsonlite::toJSON(selected) else value <- NULL
+  if (!is.null(selected)) value <- jsonlite::toJSON(gsub( "'", '"', selected)) else value <- NULL
```
Comment (jakubsob) [inline R/checkbox.R:200]: Let's remove the whitespace ;) 
```suggestion
  if (!is.null(selected)) value <- jsonlite::toJSON(gsub("'", '"', selected)) else value <- NULL
```

File: R/checkbox.R [hunk 2]
```diff
@@ -197,9 +197,9 @@ multiple_checkbox <- function(input_id, label, choices, choices_value = choices,
...
   if (!is.null(choices)) {
-    options <- jsonlite::toJSON(data.frame(name = choices, value = choices_value))
+    options <- jsonlite::toJSON(data.frame(name = choices, value = gsub( "'", '"', choices_value)))
```
Comment (jakubsob) [inline R/checkbox.R:202]: Let's remove the whitespace ;) 
```suggestion
    options <- jsonlite::toJSON(data.frame(name = choices, value = gsub("'", '"', choices_value)))
```

**Review by jakubsob — DISMISSED** [review]
Nice, it works perfectly ;) 
I just found 2 nitpicks, could we also add a test that covers this case?

**Pull request merged**